api/HTMLElement.json

```diff
@@ -1398,7 +1398,13 @@
               ]
             },
             "safari": {
-              "version_added": false
+              "version_added": "preview",
+              "flags": [
+                {
+                  "name": "inert attribute",
```

Comment:
Is it really all lowercase like this? Fine if it is, but a little surprising.

Reply:
I was surprised too but it is indeed all lower case.

![image](https://user-images.githubusercontent.com/32498324/144122337-5afe7614-a8b3-44f7-8003-7dda023c5ee1.png)

Reply:
Thanks for confirming!